scripts/build/TestPlatform.Settings.targets

```
@@ -68,6 +68,8 @@
     <When Condition="$(TestProject) == 'true'">
       <PropertyGroup>
         <GenerateDocumentationFile>false</GenerateDocumentationFile>
+        <!-- Suppress warnings about testhost being x64 (AMD64)/x86 when imported into AnyCPU (MSIL) test projects. -->
+        <NoWarn>$(NoWarn);MSB3270</NoWarn>
```

Do we need both this and the `MSBuildWarningsAsMessages` in each project?

---

We don't suppress this warning in each project, just in some. This is blanket suppression for all TestProjects.

---

Sorry I wasn't clear. I meant do we need both `NoWarn` and `MSBuildWarningsAsMessages` applied (I see that both are "suppressing" the `MSB3270`).

---

```suggestion
        <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);MSB3270</MSBuildWarningsAsMessages>
```

Do I get it correctly that you want me to do this ^^^ ?

---

Or the other way around depending on which of the 2 property has wider area but yes :)